components/event-publisher-proxy/pkg/application/clean_test.go

```diff
@@ -1,9 +1,13 @@
-package application
+//go:build unit
+// +build unit
+
+package application_test
 
 import (
 	"testing"
 
 	applicationv1alpha1 "github.com/kyma-project/kyma/components/application-operator/pkg/apis/applicationconnector/v1alpha1"
+	eppapplication "github.com/kyma-project/kyma/components/event-publisher-proxy/pkg/application"
```

Comment:
The `eppapplication` gives an impression that this is the `Event Publisher Proxy Application`. I think `application` should be more convenient here.